kubernetes/deployment.template.yaml

```diff
@@ -23,6 +23,8 @@ spec:
           value: "https://idam-web-public.aat.platform.hmcts.net"
         - name: IDAM_API_URL
           value: "https://idam-api.aat.platform.hmcts.net"
+        - name: TRIBUNALS_API_URL
+          value: "http://sscs-tribunals-api-aat.service.core-compute-aat.internal"
```

Comment:
We don't use this any more, just the charts

Reply:
thx, removed!